ipa-core/src/app.rs

```diff
@@ -251,7 +251,10 @@ impl RequestHandler<HelperIdentity> for Inner {
             }
             RouteId::CompleteQuery => {
```

Comment:
I think we also need to add `qp.complete(...)` to the Shard Handler?

Reply:
yep, this was addressed